src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/SpecialFileProviders/AbstractFindByNameUnderAppDesignerSpecialFileProvider.cs

```diff
@@ -7,11 +7,11 @@ namespace Microsoft.VisualStudio.ProjectSystem.SpecialFileProviders
     ///     that find their special file by file name under the AppDesigner folder, falling back
     ///     to the root folder if it doesn't exist.
     /// </summary>
-    internal class AbstractFindByNameUnderAppDesignerSpecialFileProvider : AbstractFindByNameSpecialFileProvider
```

Comment:
It looks like its supposed to be abstract, so looks like the fix should be mark it as such.

Reply:
Good point. We should mark this as abstract if this is not going to be instantiated. 

Should this class keep using the prefix Abstract instead of Base even if there are no abstracts methods, and the description says this is a base class ?

Reply:
Abstract and Base have the same meaning; a base class designed to be inherited from. "abstract" terminology came from the Roslyn code base which this project started from.

Reply:
Thanks for the explanation. 

It seems that "base" is more used in cps.